Command/CreateSiteCommand.php

```diff
@@ -102,9 +109,26 @@ public function execute(InputInterface $input, OutputInterface $output)
   <info>enabled</info> :  <info>from</info> {$info_enabledFrom} => <info>to</info> {$info_enabledTo}
 
 INFO
-);
+        );
 
-        if ($input->getOption('no-confirmation') || $dialog->askConfirmation($output, 'Confirm site creation ?', false)) {
+        /*
+         * NEXT_MAJOR: remove if statement and use the question helper only (true value of if) when dropping sf < 2.5
+         */
+        if ($this->getHelperSet()->has('question')) {
+            $question = new \Symfony\Component\Console\Question\ConfirmationQuestion(
+                'Confirm site creation (Y/N)',
+                false,
+                '/^(y|j)/i'
```

Comment:
If you keep it, please add `o` for french, `s` for spanish, `i` for hungarian, `h` for Klingon and so on